src/bluesky/run_engine.py

```diff
@@ -2339,9 +2340,11 @@ async def _wait(self, msg):
         if msg.args:
             (group,) = msg.args
             error_on_timeout = True
+            watch = ()
         else:
             group = msg.kwargs["group"]
             error_on_timeout = msg.kwargs.get("error_on_timeout", True)
+            watch = msg.kwargs.get("watch", ())
```

Comment:
I'm not sure I understand the argument handling of the original, would this make more sense?
```suggestion
        if msg.args:
            (group,) = msg.args
        else:
            group = msg.kwargs["group"]
        error_on_timeout = msg.kwargs.get("error_on_timeout", True)
        watch = msg.kwargs.get("watch", ())
```